modules/002-deckhouse/hooks/update_deckhouse_image.go

```diff
@@ -166,13 +166,19 @@ func updateDeckhouse(input *go_hook.HookInput, dc dependency.Container) error {
 	approvalMode := input.Values.Get("deckhouse.update.mode").String()
 	deckhouseUpdater := updater.NewDeckhouseUpdater(input, approvalMode, releaseData, deckhousePod.Ready, deckhousePod.isBootstrapImage())
 
+	labels := map[string]string{}
+	releaseChannelNameRaw, exists := input.Values.GetOk("deckhouse.releaseChannel")
+	if exists {
+		labels = map[string]string{"releaseChannel": releaseChannelNameRaw.String()}
+	}
+
 	if deckhousePod.Ready {
 		input.MetricsCollector.Expire(metricUpdatingGroup)
 		if releaseData.IsUpdating {
 			deckhouseUpdater.ChangeUpdatingFlag(false)
 		}
 	} else if releaseData.IsUpdating {
-		input.MetricsCollector.Set("d8_is_updating", 1, nil, metrics.WithGroup(metricUpdatingGroup))
+		input.MetricsCollector.Set("d8_is_updating", 1, labels, metrics.WithGroup(metricUpdatingGroup))
```

Comment:
We checked  that release channel exists [above](https://github.com/deckhouse/deckhouse/pull/3264/files#diff-a8ac2478e980525f5ff718d97d12b1d494da64b445d73a1742da319c957b4651R151).

We can simplify the code:

```
labels := map[string]string := {
   "releaseChannel": values.Get("deckhouse.releaseChannel").String() 
}

input.MetricsCollector.Set("d8_is_updating", 1, labels, metrics.WithGroup(metricUpdatingGroup))
```

Reply:
```
	labels := map[string]string{
		"releaseChannel": input.Values.Get("deckhouse.releaseChannel").String(),
	}
```
Done